src/routes/intro/IntroContainer.js

```diff
@@ -0,0 +1,186 @@
+      description: t('searchDescription'),
+      renderContent: this.renderImageContent(Search)
+    }, {
+      key: t('events'),
+      title: t('events'),
+      description: t('eventsDescription'),
+      renderContent: this.renderImageContent(Events)
+    }, {
+      key: t('offers'),
+      title: t('offers'),
+      description: t('offersDescription'),
+      renderContent: this.renderImageContent(Offers)
+    }, {
+      key: t('languageChange'),
+      title: t('languageChange'),
+      description: t('languageChangeDescription'),
+      renderContent: this.renderImageContent(Language)
+    }, {
+      key: t('inquiry'),
+      title: t('inquiryTitle'),
+      renderContent: this.renderSettings
+    }]
+  }
+
+  setAllowPushNotifications = () => this.setState(prevState =>
+    ({ allowPushNotifications: !prevState.allowPushNotifications }))
+
+  setUseLocationAccess = () => this.setState(prevState => ({ useLocationAccess: !prevState.useLocationAccess }))
+
+  setAllowSentry = () => this.setState(prevState => ({ allowSentry: !prevState.allowSentry }))
+
+  renderSettings = (): React.Node => {
+    const { t, theme } = this.props
+    const themeColor = theme.colors.themeColor
+    const { allowPushNotifications, useLocationAccess } = this.state
+
+    return <>
+      <SettingItem bigTitle title={t('pushNewsTitle')} description={t('pushNewsDescription')}
+                   onPress={this.setAllowPushNotifications} theme={theme}>
+        <Switch thumbColor={themeColor} trackColor={{ true: themeColor }}
+                onValueChange={this.setAllowPushNotifications} value={allowPushNotifications} />
+      </SettingItem>
+      <SettingItem bigTitle title={t('locationTitle')} description={t('locationDescription')}
+                   onPress={this.setUseLocationAccess} theme={theme}>
+        <Switch thumbColor={themeColor} trackColor={{ true: themeColor }}
+                onValueChange={this.setUseLocationAccess} value={useLocationAccess} />
+      </SettingItem>
+      <SettingItem bigTitle title={t('sentryTitle')} description={t('sentryDescription')}
+                   onPress={this.setAllowSentry} theme={theme}>
+        <Switch thumbColor={themeColor} trackColor={{ true: themeColor }}
+                onValueChange={this.setAllowSentry} value={useLocationAccess} />
+      </SettingItem>
+    </>
+  }
+
+  onAccept = () => { this.onDone(true, true, true) }
+
+  onContinue = () => {
+    const { allowPushNotifications, useLocationAccess, allowSentry } = this.state
+    this.onDone(allowPushNotifications, allowSentry, useLocationAccess)
+  }
+
+  onDone = async (allowPushNotifications: boolean, errorTracking: boolean, useLocationAccess: boolean) => {
+    try {
+      if (errorTracking) {
+        const sentry = new SentryIntegration()
+        await sentry.install()
+      }
+
+      if (useLocationAccess) {
+        // TODO request permission, return if not granted
+      }
+    } catch (e) {
+      console.warn(e)
```

Comment:
Is this required because of Sentry?

Reply:
I think so, yes. Was requested by @Schedulaar